aws_cloudfront_distribution_tenant: panic on create when managed certificate is enabled

### Terraform and AWS Provider Version

```console
Terraform v1.14.6
provider registry.terraform.io/hashicorp/aws v6.35.0
```

### Affected Resource(s) or Data Source(s)

*`aws_cloudfront_distribution_tenant`

### Expected Behavior

When creating an aws_cloudfront_distribution_tenant with `managed_certificate_request` and `wait_for_deployment = true` the creation of the distribution should succeed without errors

### Actual Behavior

The provider crashes with a nil pointer dereference panic in updateDistributionTenantWithManagedCertificate at distribution_tenant.go:841. after the distribution is created, the state isnt' saved even though the resource is created on aws, likely because the api call succeeds but the wait for certificate does not



### Relevant Error/Panic Output

```console
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x17cfd24a]

goroutine 119 [running]:
github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.updateDistributionTenantWithManagedCertificate({0x22ee1bb8?, 0xc00245ebd0?}, 0xc002012388?, 0xc0017bab00?, 0x1e?)
    github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:841 +0x2a
github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.waitManagedCertificateReady({0x22ee1bb8, 0xc00245ebd0}, 0xc002012388, {0xc0017bab00, 0x1e}, 0xc0039561f8?)
    github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:784 +0x1c5
github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.(*distributionTenantResource).Create(...)
    github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:329 +0xa1c

```


### Sample Terraform Configuration

<details open>
<summary>Click to expand configuration</summary>

```hcl

resource "aws_cloudfront_distribution_tenant" "example" {
  name            = "example-tenant"
  distribution_id = aws_cloudfront_multitenant_distribution.example.id
  enabled         = true

  domain {
    domain = "tenant.example.com"
  }

  managed_certificate_request {
    certificate_transparency_logging_preference = "enabled"
    primary_domain_name                         = "tenant.example.com"
  }

  wait_for_deployment = true
}
```
</details>


### Steps to Reproduce

1. Create an `aws_cloudfront_multitenant_distribution`
2. Create an `aws_cloudfront_distribution_tenant` with `managed_certificate_request` and `wait_for_deployment = true`
3. Run `terraform apply`
4. The provider panics with a SIGSEGV

### Debug Logging

<details open>
<summary>Click to expand log output</summary>

```console
2026-03-06T06:23:07.898+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: request failed with unretryable error https response error StatusCode: 404, RequestID: 99f90cee-f713-44f5-b9ba-f1b876a6e7f0, EntityNotFound: The specified CloudFront managed ACM certificate doesn't exist. Specify another certificate and try again.: rpc.system.name=aws-api tf_req_id=bc3a2ed4-1ded-d641-5d44-fae8a86ae972 @module=aws tf_mux_provider="*proto5server.Server" tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_cloudfront_distribution_tenant tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.71/logging/tf_logger.go:45 rpc.method=GetManagedCertificateDetails tf_aws.sdk=aws-sdk-go-v2 aws.region=us-east-1 timestamp="2026-03-06T06:23:07.898+0100"
  2026-03-06T06:23:07.898+0100 [TRACE] provider.terraform-provider-aws_v6.35.0_x5: Served request: tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=bc3a2ed4-1ded-d641-5d44-fae8a86ae972 tf_resource_type=aws_cloudfront_distribution_tenant tf_rpc=ApplyResourceChange @caller=runtime/panic.go:783 tf_proto_version=5.11 @module=sdk.proto timestamp="2026-03-06T06:23:07.898+0100"
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: panic: runtime error: invalid memory address or nil pointer dereference
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x17cfd24a]
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: goroutine 102 [running]:
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.updateDistributionTenantWithManagedCertificate({0x22ee1bb8?, 0xc001ef00c0?}, 0xc001f16008?, 0xc001d714e0?, 0x1e?)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:841 +0x2a
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.waitManagedCertificateReady({0x22ee1bb8, 0xc001ef00c0}, 0xc001f16008, {0xc001d714e0, 0x1e}, 0xc00242c108?)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:784 +0x1c5
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.(*distributionTenantResource).Create(0xc0019db7d0, {0x22ee1bb8, 0xc001ef00c0}, {{{{0x22ff1f40, 0xc001eb46c0}, {0x1e9b9380, 0xc001e82240}}, {0x23054718, 0xc0017aefa0}}, {{{0x22ff1f40, ...}, ...}, ...}, ...}, ...)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:329 +0xa1c
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-provider-aws/internal/provider/framework.(*wrappedResource).Create.interceptedHandler[...].func1(...)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-provider-aws/internal/provider/framework/intercept.go:403
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-provider-aws/internal/provider/framework.(*wrappedResource).Create(0xc004478b90, {0x22ee1bb8?, 0xc0019db770?}, {{{{0x22ff1f40, 0xc001eb46c0}, {0x1e9b9380, 0xc001e82240}}, {0x23054718, 0xc0017aefa0}}, {{{0x22ff1f40, ...}, ...}, ...}, ...}, ...)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-provider-aws/internal/provider/framework/wrap.go:684 +0x3e3
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).CreateResource(0xc002ad0708, {0x22ee1bb8, 0xc0019db770}, 0xc0015ff470, 0xc0015ff440)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-plugin-framework@v1.18.0/internal/fwserver/server_createresource.go:127 +0x88e
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ApplyResourceChange(0xc002ad0708, {0x22ee1bb8, 0xc0019db770}, 0xc000e23500, 0xc0015ff598)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-plugin-framework@v1.18.0/internal/fwserver/server_applyresourcechange.go:63 +0x586
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ApplyResourceChange(0xc002ad0708, {0x22ee1bb8?, 0xc0019db680?}, 0xc004478af0)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-plugin-framework@v1.18.0/internal/proto5server/server_applyresourcechange.go:71 +0x585
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ApplyResourceChange(0xc00355b7a0, {0x22ee1bb8?, 0xc0019db320?}, 0xc004478af0)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-plugin-mux@v0.22.0/tf5muxserver/mux_server_ApplyResourceChange.go:36 +0x18d
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc003530e60, {0x22ee1bb8?, 0xc0019da840?}, 0xc0038af100)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-plugin-go@v0.30.0/tfprotov5/tf5server/server.go:944 +0x3b9
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x204ae700, 0xc003530e60}, {0x22ee1bb8, 0xc0019da840}, 0xc0038af080, 0x0)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	github.com/hashicorp/terraform-plugin-go@v0.30.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:789 +0x1a6
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: google.golang.org/grpc.(*Server).processUnaryRPC(0xc00350b448, {0x22ee1bb8, 0xc0019da7b0}, 0xc002ae21a0, 0xc0035fcb70, 0x323c7368, 0x0)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	google.golang.org/grpc@v1.79.1/server.go:1426 +0x1192
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: google.golang.org/grpc.(*Server).handleStream(0xc00350b448, {0x22ee2678, 0xc002b3e000}, 0xc002ae21a0)
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	google.golang.org/grpc@v1.79.1/server.go:1830 +0xdc6
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: google.golang.org/grpc.(*Server).serveStreams.func2.1()
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	google.golang.org/grpc@v1.79.1/server.go:1061 +0x7f
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 36
  2026-03-06T06:23:07.900+0100 [DEBUG] provider.terraform-provider-aws_v6.35.0_x5: 	google.golang.org/grpc@v1.79.1/server.go:1072 +0x11d
  2026-03-06T06:23:07.909+0100 [ERROR] provider: plugin process exited: plugin=.terraform/providers/registry.terraform.io/hashicorp/aws/6.35.0/linux_amd64/terraform-provider-aws_v6.35.0_x5 id=1302178 error="exit status 2"
  2026-03-06T06:23:07.909+0100 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
  2026-03-06T06:23:07.922+0100 [ERROR] plugin.(*GRPCProvider).ApplyResourceChange: error="rpc error: code = Unavailable desc = error reading from server: EOF"
  2026-03-06T06:23:07.922+0100 [TRACE] maybeTainted: aws_cloudfront_distribution_tenant.this encountered an error during creation, so it is now marked as tainted
  2026-03-06T06:23:07.922+0100 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/hashicorp/aws" is in the global cache
  2026-03-06T06:23:07.922+0100 [TRACE] NodeAbstractResouceInstance.writeResourceInstanceState to workingState for aws_cloudfront_distribution_tenant.this
  2026-03-06T06:23:07.922+0100 [TRACE] NodeAbstractResouceInstance.writeResourceInstanceState: removing state object for aws_cloudfront_distribution_tenant.this
  2026-03-06T06:23:07.922+0100 [TRACE] evalApplyProvisioners: aws_cloudfront_distribution_tenant.this is tainted, so skipping provisioning
  2026-03-06T06:23:07.922+0100 [TRACE] maybeTainted: aws_cloudfront_distribution_tenant.this was already tainted, so nothing to do
  2026-03-06T06:23:07.922+0100 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/hashicorp/aws" is in the global cache
  2026-03-06T06:23:07.922+0100 [TRACE] NodeAbstractResouceInstance.writeResourceInstanceState to workingState for aws_cloudfront_distribution_tenant.this
  2026-03-06T06:23:07.922+0100 [TRACE] NodeAbstractResouceInstance.writeResourceInstanceState: removing state object for aws_cloudfront_distribution_tenant.this
  2026-03-06T06:23:07.922+0100 [DEBUG] State storage *remote.State declined to persist a state snapshot
  2026-03-06T06:23:07.922+0100 [ERROR] vertex "aws_cloudfront_distribution_tenant.this" error: Plugin did not respond
  2026-03-06T06:23:07.922+0100 [TRACE] vertex "aws_cloudfront_distribution_tenant.this": visit complete, with errors
  2026-03-06T06:23:07.922+0100 [TRACE] dag/walk: upstream of "output.tenant (expand)" errored, so skipping
  2026-03-06T06:23:07.922+0100 [TRACE] dag/walk: upstream of "provider[\"registry.terraform.io/hashicorp/aws\"] (close)" errored, so skipping
  2026-03-06T06:23:07.923+0100 [TRACE] dag/walk: upstream of "root" errored, so skipping
  2026-03-06T06:23:07.923+0100 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/hashicorp/aws" is in the global cache
  2026-03-06T06:23:07.923+0100 [DEBUG] states/remote: state read serial is: 1; serial is: 1
  2026-03-06T06:23:07.923+0100 [DEBUG] states/remote: state read lineage is: 05fcd971-0682-7f16-4725-deaeaa5fcc24; lineage is: 05fcd971-0682-7f16-4725-deaeaa5fcc24

Stack trace from the terraform-provider-aws_v6.35.0_x5 plugin:
  
  panic: runtime error: invalid memory address or nil pointer dereference
  [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x17cfd24a]
  
  goroutine 102 [running]:
  github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.updateDistributionTenantWithManagedCertificate({0x22ee1bb8?, 0xc001ef00c0?}, 0xc001f16008?, 0xc001d714e0?, 0x1e?)
  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:841 +0x2a
  github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.waitManagedCertificateReady({0x22ee1bb8, 0xc001ef00c0}, 0xc001f16008, {0xc001d714e0, 0x1e}, 0xc00242c108?)
  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:784 +0x1c5
  github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront.(*distributionTenantResource).Create(0xc0019db7d0, {0x22ee1bb8, 0xc001ef00c0}, {{{{0x22ff1f40, 0xc001eb46c0}, {0x1e9b9380, 0xc001e82240}}, {0x23054718, 0xc0017aefa0}}, {{{0x22ff1f40, ...}, ...}, ...}, ...}, ...)
  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront/distribution_tenant.go:329 +0xa1c
  github.com/hashicorp/terraform-provider-aws/internal/provider/framework.(*wrappedResource).Create.interceptedHandler[...].func1(...)
  	github.com/hashicorp/terraform-provider-aws/internal/provider/framework/intercept.go:403
  github.com/hashicorp/terraform-provider-aws/internal/provider/framework.(*wrappedResource).Create(0xc004478b90, {0x22ee1bb8?, 0xc0019db770?}, {{{{0x22ff1f40, 0xc001eb46c0}, {0x1e9b9380, 0xc001e82240}}, {0x23054718, 0xc0017aefa0}}, {{{0x22ff1f40, ...}, ...}, ...}, ...}, ...)
  	github.com/hashicorp/terraform-provider-aws/internal/provider/framework/wrap.go:684 +0x3e3
  github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).CreateResource(0xc002ad0708, {0x22ee1bb8, 0xc0019db770}, 0xc0015ff470, 0xc0015ff440)
  	github.com/hashicorp/terraform-plugin-framework@v1.18.0/internal/fwserver/server_createresource.go:127 +0x88e
  github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ApplyResourceChange(0xc002ad0708, {0x22ee1bb8, 0xc0019db770}, 0xc000e23500, 0xc0015ff598)
  	github.com/hashicorp/terraform-plugin-framework@v1.18.0/internal/fwserver/server_applyresourcechange.go:63 +0x586
  github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ApplyResourceChange(0xc002ad0708, {0x22ee1bb8?, 0xc0019db680?}, 0xc004478af0)
  	github.com/hashicorp/terraform-plugin-framework@v1.18.0/internal/proto5server/server_applyresourcechange.go:71 +0x585
  github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ApplyResourceChange(0xc00355b7a0, {0x22ee1bb8?, 0xc0019db320?}, 0xc004478af0)
  	github.com/hashicorp/terraform-plugin-mux@v0.22.0/tf5muxserver/mux_server_ApplyResourceChange.go:36 +0x18d
  github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc003530e60, {0x22ee1bb8?, 0xc0019da840?}, 0xc0038af100)
  	github.com/hashicorp/terraform-plugin-go@v0.30.0/tfprotov5/tf5server/server.go:944 +0x3b9
  github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x204ae700, 0xc003530e60}, {0x22ee1bb8, 0xc0019da840}, 0xc0038af080, 0x0)
  	github.com/hashicorp/terraform-plugin-go@v0.30.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:789 +0x1a6
  google.golang.org/grpc.(*Server).processUnaryRPC(0xc00350b448, {0x22ee1bb8, 0xc0019da7b0}, 0xc002ae21a0, 0xc0035fcb70, 0x323c7368, 0x0)
  	google.golang.org/grpc@v1.79.1/server.go:1426 +0x1192
  google.golang.org/grpc.(*Server).handleStream(0xc00350b448, {0x22ee2678, 0xc002b3e000}, 0xc002ae21a0)
  	google.golang.org/grpc@v1.79.1/server.go:1830 +0xdc6
  google.golang.org/grpc.(*Server).serveStreams.func2.1()
  	google.golang.org/grpc@v1.79.1/server.go:1061 +0x7f
  created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 36
  	google.golang.org/grpc@v1.79.1/server.go:1072 +0x11d
  
  Error: The terraform-provider-aws_v6.35.0_x5 plugin crashed!
  
  This is always indicative of a bug within the plugin. It would be immensely
  helpful if you could report the crash with the plugin's maintainers so that it
  can be fixed. The output above should help diagnose the issue.
  
  2026-03-06T06:23:08.695+0100 [DEBUG] provider: plugin exited
  
  exit status 1


```
</details>


### GenAI / LLM Assisted Development

Claude Code

### Important Facts and References

_No response_

### Would you like to implement a fix?

Yes

--- a/.changelog/46982.txt
+++ b/.changelog/46982.txt
@@ -1,0 +1,3 @@
+```release-note:bug
+resource/aws_cloudfront_distribution_tenant: Fix panic when managed certificate is not found during creation
+```

--- a/internal/service/cloudfront/cloudfront_test.go
+++ b/internal/service/cloudfront/cloudfront_test.go
@@ -18,5 +18,6 @@ func init() {
 func testAccErrorCheckSkipFunction(t *testing.T) resource.ErrorCheckFunc {
 	return acctest.ErrorCheckSkipMessagesContaining(t,
 		"InvalidParameterValueException: Unsupported source arn",
+		"no matching Route 53 Hosted Zone found",
 	)
 }

--- a/internal/service/cloudfront/distribution_tenant.go
+++ b/internal/service/cloudfront/distribution_tenant.go
@@ -837,6 +837,11 @@ func waitForManagedCertificateIssued(ctx context.Context, conn *cloudfront.Clien
 }
 
 func updateDistributionTenantWithManagedCertificate(ctx context.Context, conn *cloudfront.Client, dtOutput *cloudfront.GetDistributionTenantOutput, mcOutput *cloudfront.GetManagedCertificateDetailsOutput) error {
+	if mcOutput == nil {
+		// No managed certificate found, nothing to update
+		return nil
+	}
+
 	// Check if we need to update the certificate ARN
 	if !needToUpdateCertificateARN(dtOutput.DistributionTenant, aws.ToString(mcOutput.ManagedCertificateDetails.CertificateArn)) {
 		// Certificate ARN already matches, nothing to do

--- a/internal/service/cloudfront/distribution_tenant.go
+++ b/internal/service/cloudfront/distribution_tenant.go
@@ -838,7 +838,6 @@ func waitForManagedCertificateIssued(ctx context.Context, conn *cloudfront.Clien
 
 func updateDistributionTenantWithManagedCertificate(ctx context.Context, conn *cloudfront.Client, dtOutput *cloudfront.GetDistributionTenantOutput, mcOutput *cloudfront.GetManagedCertificateDetailsOutput) error {
 	if mcOutput == nil {
-		// No managed certificate found, nothing to update
 		return nil
 	}
 

